Source/SwiftLintBuiltInRules/Rules/Lint/UnusedDeclarationRuleExamples.swift

```diff
@@ -44,6 +44,15 @@ struct UnusedDeclarationRuleExamples {
         }
         _ = ResponseModel()
         """),
+        Example("""
```

Comment:
If this test fails on Swift < 5.8, you can conditionally add this example only when compiling with 5.8+ like we do for other rules.